src/MIDebugEngine/AD7.Impl/AD7Property.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using EnvDTE;
```

Comment:
What is using this? I don't think you want this. If so, we probably need to ship another dll.